OpenRA.Mods.Common/Effects/FloatingSprite.cs

```diff
@@ -15,7 +15,7 @@
 
 namespace OpenRA.Mods.Common.Effects
 {
-	sealed class FloatingSprite : IEffect, ISpatiallyPartitionable
+	public sealed class FloatingSprite : IEffect, ISpatiallyPartitionable
```

Comment:
```suggestion
	public class FloatingSprite : IEffect, ISpatiallyPartitionable
```

Does it need to remain sealed?

Reply:
I don't know the reason it has been sealed in the first place, so I can't say. All fields are private, so currently no customization is possible. `Mine` class [remained sealed](https://github.com/OpenRA/OpenRA/commit/22b39f35aa6242c636cf996d3532ccbaadbd4d21) after making it public (again no extension points as all methods are implemented explicitly).